CONTRIBUTING.md

```diff
@@ -7,4 +7,15 @@ Please first discuss the change you wish to make via an issue before making a ch
 1. Ensure that there are automated tests that cover any changes 
 1. Update the README.md with details of any significant changes to functionality
 1. Ensure that your commit messages increment the version using [GitVersion syntax](https://gitversion.readthedocs.io/en/latest/input/docs/more-info/version-increments/). If no message is found then the patch version will be incremented by default.
-1. You may merge the pull request once it meets all of the required checks. If you do not have permision, a reviewer will do it for you
\ No newline at end of file
+1. You may merge the pull request once it meets all of the required checks. If you do not have permision, a reviewer will do it for you
+
+## Lifecycle Call
```

Comment:
Might be better to link to [Define custom code for your package](https://docs.microsoft.com/en-us/power-platform/alm/package-deployer-tool#step-5-define-custom-code-for-your-package) as this describes when each of these methods are called in detail.

Reply:
I'll certainly add this link, that's where I got my source of information but I found it still a little confusing what was called when. This section just lays that out in the simplest form.